src/engine/controls/enginecontrol.h

```diff
@@ -69,6 +69,7 @@ class EngineControl : public QObject {
     virtual void notifySeek(double dNewPlaypos);
     virtual void trackLoaded(TrackPointer pNewTrack);
     virtual void trackBeatsUpdated(mixxx::BeatsPointer pBeats);
+    virtual void trackUnloaded(TrackPointer pOldTrack);
```

Comment:
I suggest, to keep trackLoaded/trackUnloaded adjacent to each other to prevent someone from reordering this stuff again in the future.